native-server/src/main/java/com/datastax/oss/simulacron/server/BoundNode.java

```diff
@@ -431,13 +432,15 @@ void handle(ChannelHandlerContext ctx, Frame frame) {
       } else if (frame.message instanceof Execute) {
         // Unprepared execute received, return an unprepared.
         Execute execute = (Execute) frame.message;
+        logger.warn("No stub mapping found for message type EXECUTE: \"{}\"", execute.toString());
         String hex = new BigInteger(1, execute.queryId).toString(16);
         response = new Unprepared("No prepared statement with id: " + hex, execute.queryId);
       } else if (frame.message instanceof Prepare) {
         // fake up a prepared statement from the message and register an internal prime for it.
         Prepare prepare = (Prepare) frame.message;
         // TODO: Maybe attempt to identify bind parameters
         String query = prepare.cqlQuery;
+        logger.info("No stub mapping found for message type PREPARE: \"{}\". Registering priming...", query);
```

Comment:
Why info for this one?

Reply:
I would assume a mismatch on a prepared statement would occur much more often. Also, I would expect devs to not prime all prepared statements. That's why I was thinking of having a lower log level for this one. Thinking of it more, we could even lower it down to debug. 

What are your thoughts on this?

Reply:
OK, makes sense, let's go with with info.